src/data/faq.json

```diff
@@ -324,6 +324,26 @@
                             "The digital proof of vaccination is generated in the doctor's practice or in a vaccination center. After entering or transferring the data, a vaccination certificate token (QR code) will be created for you, which you can scan directly with the Corona-Warn-App (CWA) or can be handed to you on a paper printout and later scanned with the CWA.  You can then manage your digital vaccination certificates on your smartphone."
                         ]
                     },
+                    {
+                        "title": "How do I become aware of booster vaccinations in the context of Corona?",
+                        "anchor": "vac_cert_booster_info",
+                        "active": true,
+                        "textblock": [
+                            "The Conference of Health Ministers has now passed a regulation on COVID 19 booster vaccinations. An additional booster vaccination is designed to help raise neutralizing antibody titers again, which may decline over time after basic immunization. This becomes particularly important if a weak/muted vaccine response is assumed or recognized after basic immunization in certain groups of vulnerable people.",
+                            "The Corona Warn App (CWA) will notify the user to consider a booster vaccination if certain conditions are met. These conditions are based on the recommendations of the Paul Ehrlich Institute and the Permanent Vaccination Commission (STIKO) of the Robert Koch Institute (RKI).",

```

Comment:
```suggestion
                            "The Corona-Warn-App (CWA) will notify the user to consider a booster vaccination if certain conditions are met. These conditions are based on the recommendations of the Paul Ehrlich Institute and the Permanent Vaccination Commission (STIKO) of the Robert Koch Institute (RKI).",
```
Corona-Warn-App uses hyphens also in English - see https://www.coronawarn.app/en/

Reply:
Committed your suggestion, thanks!